index.html

```diff
@@ -12376,6 +12608,14 @@ <h2>Relationship to RDF</h2>
       and <code>false</code>. The JSON-LD 1.1 Processing Algorithms and API specification [[JSON-LD11-API]]
       defines the <a data-cite="JSON-LD11-API#data-round-tripping">conversion rules</a>
       between JSON's native data types and RDF's counterparts to allow round-tripping.</li>
+    <li class="changed">As an extension to the <a data-cite="RDF11-CONCEPTS#data-model">RDF data model</a>,
+      <em>typed literals</em>, where the datatype is `xsd:string`, or <a>language-tagged strings</a>
+      MAY include a <a>base direction</a>.
+      As there is currently no standardized mechanism for representing the <a>base direction</a>
+      of <a>RDF literals</a>, the JSON-LD to RDF transformation loses the base direction.
```

Comment:
Given the following sentence, this one is a bit confusing. Perhaps a slight adjustment:

```suggestion
      of <a>RDF literals</a>, the JSON-LD to standard RDF transformation loses the base direction.
```